packages/notifications/lib/utils.js

```diff
@@ -0,0 +1,11 @@
+const deleteSessionForDevices = async (tokens, pgdb) => {
+  return pgdb.query(`
+    DELETE FROM sessions s WHERE s.id IN (
+                                         SELECT d."sessionId" FROM devices d WHERE d.id IN (:tokens)
```

Comment:
I wonder if/why `IN (:tokens)` works. Pogi says you should use `&&` here: https://pogi.readthedocs.io/en/latest/pitfalls/#named-parameter-column-in-expression

Reply:
Maybe because of the parens around `:tokens`?